src/tools/histogram_dump.py

```diff
@@ -55,13 +62,34 @@ def print_histogram(asok, logger, counter, last):
     content += ("{0: >"+str(len(axes[1]['ranges'])*5+14)+"}:\n").format(
         axes[0]['name'])
 
+    COL = '\033[91m'
```

Comment:
Can we display colors only when `sys.stdout.isatty()` returns True? May be useful for scripting.
Forcing disabled/enabled colors could also be an option passed to the script.

Reply:
Added with https://github.com/vico-research-and-consulting/ceph/commit/9d5bb0c616b07dae937c3f68afd30cc366832199 this new feature :-)